### PR TITLE
add elixir lang test - uses binary encoding

### PR DESCRIPTION
**Pull request by mpisanko**

### CODE REVIEW

File: lang/elixr/lib/elixr.ex [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  ## Examples
+
+      iex> Elixr.hello()
+      :world
+
+  """
+  def hello do
+    :world
+  end
+end
```
Comment (FiV0) [inline lang/elixr/lib/elixr.ex]: What is this file for?

File: core/src/main/clojure/xtdb/types.clj [hunk 2]
```diff
@@ -818,13 +825,14 @@
         oid (.getInt bb 12)
         length (.getInt bb 16)
         [typlen get-elem] (case oid
-                            (23 1007) [4 (fn [bb idx] (.getInt bb idx))]
-                            (20 1016) [8 (fn [bb idx] (.getLong bb idx))])
-        start  (+ data-offset 20)]
+                            (23 1007) [4 (fn [bb ^long start ^long typlen ^long idx] (.getInt bb (+ start (* idx typlen))))]
+                            (20 1016) [8 (fn [bb ^long start ^long typlen ^long idx] (.getLong bb (+ start (* idx typlen))))])
```
Comment (FiV0) [inline core/src/main/clojure/xtdb/types.clj]: These function still has reflection warnings on the `.getInt` and `.getLong`.

**Review by FiV0 — CHANGES_REQUESTED** [review]
Just two tiny remarks otherwise LGTM.

File: lang/elixr/lib/elixr.ex (hunk 1 above)
Comment (mpisanko) [reply lang/elixr/lib/elixr.ex]: removed- not needed

**Pull request merged**